src/arm/midr.c

```diff
@@ -21,6 +25,66 @@
 #include "uarch.h"
 #include "sve.h"
 
+
+#if defined _WIN32
+// Windows stores processor information in registery at:
+// "HKEY_LOCAL_MACHINE\HARDWARE\DESCRIPTION\System\CentralProcessor"
+// Within this directory, there each core will get its own folder as well with
```

Comment:
`Within this directory, there each core` -> `Within this directory, each core...`